### DISCUSSION TITLE
Would it be possible to not show the "people also ask" and "related searches" sections in the results? it is so annoying and the primary reason I even moved away from google, I just want search results when I search for something.

### DISCUSSION THREAD

**Discussion by ghost** [benbusby/whoogle-search#541] [Ideas]
I've deployed my instance with heroku and I have searched around quite a bit finding no helpful info, please tell me how to do this if possible or introduce it as a new feature. If necessary I will open a new issue but I just wanted to confirm it once in discussions.

As always I love your project and look ahead to using it and maybe even contributing when I'm competent enough (sometime in the future) 😀

**Comment by DUOLabs333** [comment]
There's actually a `WHOOGLE_MINIMAL_MODE` for exactly this.
